sdcm/utils/version_utils.py

```diff
@@ -71,3 +77,12 @@ def is_enterprise(version):
     :return: True if this version string passed is a scylla enterprise version
     """
     return parse_version(version) > parse_version('2000')
+
+
+def get_systemd_version(output: str) -> int:
+    if match := SYSTEMD_VERSION_RE.match(output):
+        try:
+            return int(match.group("version"))
+        except ValueError:
+            pass
+    return 0
```

Comment:
missing an empty line in the end of this file

Reply:
don't care..., this branch is breaking all of the checks we have anyhow..